R/control.R

```diff
@@ -52,6 +52,8 @@ spim_control <- function(short_run, n_chains, deterministic = FALSE,
     burnin <- burnin
   }
 
+  n_steps_retain <- ceiling(n_sample / n_chains)
```

Comment:
Is this definitely the easiest way think about it? Would you not normally expect the number of chains to increase this?

Reply:
This is the number of steps retain per chain, right? So it makes sense that more chains means you reduce the number of steps retained per chain

Reply:
The other way of thinking about it would be that n_sample is the number to keep per chain.  There's certainly no wrong way, just what would be more or less surprising to you or someone else when changing the number of chains